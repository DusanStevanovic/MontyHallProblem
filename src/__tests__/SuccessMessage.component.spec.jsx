// @flow
import React from 'react';
import { shallow } from 'enzyme';
import SuccessMessage from '../components/ErrorMessage.component';

const mockProps = {
  message: ['Success !']
}

describe('SuccessMessage', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SuccessMessage { ...mockProps } />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should display text from props', () => {
    const wrapper = shallow(<SuccessMessage { ...mockProps } />);

    expect(wrapper.text()).toEqual(mockProps.message[0]);
  });
});