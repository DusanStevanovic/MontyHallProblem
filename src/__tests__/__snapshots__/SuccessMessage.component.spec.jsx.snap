// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessMessage should match snapshot 1`] = `ShallowWrapper {}`;
